src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/ECDsa/ECDsaTests.netcoreapp.cs

```diff
@@ -227,6 +227,7 @@ public void TestRegenKeyNistP256()
 
         [Theory]
         [MemberData(nameof(TestCurves))]
+        [SkipOnMono("Checking if a curve is named is unsupported on Android", TestPlatforms.Android)]
```

Comment:
Didn't you get this working?

Reply:
I did. I just forgot to re-enable the test.